rainbowstream/rainbow.py

```diff
@@ -330,7 +330,10 @@ def home():
     num = c['HOME_TWEET_NUM']
     if g['stuff'].isdigit():
         num = int(g['stuff'])
-    for tweet in reversed(t.statuses.home_timeline(count=num)):
+    kwargs = {'count': num}
+    if not c.get('DISABLE_EXTENDED_TWEETS'):
```

Comment:
It would be nicer to create a common function to make the code looks more DRY. util.py should be a good place
```python
def add_tweetmode_parameter(kwargs):
    if not c.get('DISABLE_EXTENDED_TWEETS'):
        kwargs['tweet_mode'] = 'extended'
    return kwargs
```

Reply:
You're absolutely right :+1: . This modification has been added to my branch.